modules/sdk/gradle-plugins-workspace/CHANGELOG.markdown

```diff
@@ -223,6 +223,13 @@ version 3.2.5.
 - [LPS-82534]: Update the [Liferay Gradle Plugins] dependency to version
 3.12.46.
 
+## 1.10.2 - 2018-06-18
+- [LPS-74544]: Updated dependency to [Liferay Gradle Plugins] to version
```

Comment:
Please be consistent with the rest of the change log. It's `Update the...`, not `Updated dependency to`.